src/javascript/app_2/components/elements/data_table.jsx

```diff
@@ -86,7 +86,7 @@ class DataTable extends React.PureComponent {
 DataTable.propTypes = {
     columns         : PropTypes.array,
     data_source     : PropTypes.array,
-    footer          : PropTypes.string,
+    footer          : PropTypes.any, // footer can be object || string
```

Comment:
I think footer is passing `DOM` as a footer. The purpose of writing PropTypes is to restrict the prop type so that we can further make the testing and verification easy. 

Solution
1) Try use `console.log(footer)` and refactor `footer` prop as much as you can
2) If there is no other way to change `footer` prop, restrict the `footer` prop to two types. `footer: PropTypes.oneOfTypes([PropTypes.object, PropTypes.string])`.

Thanks!